Description: Fix a typo in the mpiexec man page
 Fix a typo in the mpiexec perl wrapper that produces the corresponding
 man page
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=10140

--- a/contribs/torque/mpiexec.pl
+++ b/contribs/torque/mpiexec.pl
@@ -178,7 +178,7 @@
 
 where args are comannd line arguments for mpiexec (see below), executable is
 the name of the eecutable and pgmargs are command line arguments for the
-executable. For example the following command will run the MPI progam a.out on
+executable. For example the following command will run the MPI program a.out on
 4 processes:
 
 	mpiexec.slurm -n 4 a.out
